npc/re/merchants/barters.yml

```
@@ -44,3 +44,579 @@
 Header:
   Type: BARTER_DB
   Version: 1
+
```

```suggestion

Footer:
  Imports:
  - Path: npc/re/merchants/barters/eden_market.yml
```
then add the content of this file into `eden_market.yml` to organize the barter shops